modules/cloud-config-container/simple-nva/cloud-config.yaml

```diff
@@ -43,6 +43,12 @@ write_files:
 %{ if enable_health_checks ~}
```

Comment:
can you please add some spaces at least between the main logical blocks. It would help with the readability

Reply:
can we also update the copyright year at the beginning of the file?

Reply:
Updated copyright, do you mean adding whitespaces to indent based on login (not sure this is feasible) or new line to separate sections?

Reply:
yes, new lines if possible.

Reply:
Added new lines (tested) to separated file sections.